packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -122,7 +123,8 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
-            history.push(routes.cashier_withdrawal);
+            const is_next_wallet = localStorage.getObject('FeatureFlagsStore')?.data.next_wallet;
+            is_next_wallet ? history.push(routes.wallets_withdrawal) : history.push(routes.cashier_withdrawal);
```

Comment:
```suggestion
            const is_next_wallet = localStorage.getObject('FeatureFlagsStore')?.data.next_wallet;
            is_next_wallet ? history.push(routes.wallets_withdrawal) : history.push(routes.cashier_withdrawal);
```
Does this needs to be enabled at this stage?

Reply:
The reason for this check is to make the url from the email redirect to the wallets cashier withdrawal page instead of the legacy cashier withdrawal page when the next wallet feature flag is enabled. `is_next_wallet` is however moved out of the case as per @thisyahlen-deriv's [review](https://github.com/binary-com/deriv-app/pull/11081#discussion_r1378350126)